train.py

```diff
@@ -170,6 +170,18 @@ def validate_directories(args):
     }
 
 
+def check_sample_size(dilations, sample_size):
+    '''Check that the sample size is greater than the receptive field size and
+       raise an exception if not.'''
+
+    receptive_field_size = compute_receptive_field_size(dilations)
+    if sample_size is not None:
+        if sample_size <= receptive_field_size:
+            raise ValueError('Sample size {} must be greater than the '
+                             'receptive field size {}.'. \
```

Comment:
These kind of line breaks are very ugly, I'm surprised pep8 hasn't complained about it. I would put `format(` here and the arguments in the next line if they don't fit, or something like that.